en/docs/install-and-setup/setup/distributed-deployment/synchronizing-artifacts-in-a-gateway-cluster.md

```diff
@@ -157,6 +157,12 @@ password = "wso2carbon"
 Add the tables of the `AM_GW_API_ARTIFACTS` and `AM_GW_PUBLISHED_API_DETAILS` to this new database that you are specifying. 
 The scripts to create these tables are in the `<API-M_HOME>/dbscripts/apimgt/` directory.
 
+!!! important
+
+     **Migrating from file based mechanism to inbuilt artifact synchronization**
+
+      * When transitioning to database (DB) retrieval mode, it is essential not to use two storage methods simultaneously to avoid potential issues.
```

Comment:
`DB retrieval mode` --> Does this mean data_retrieval_mode configuration?

Reply:
yes,  it refers to the configuration we need to add to store artifacts in db

Reply:
My concern was we haven't mentioned about a DB retrieval mode before. May be we can use the term relating it with the `data_retrieval_mode config`. Then it would be clear.